app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime

```

Comment:
Ensure that the import statements are grouped and ordered correctly. According to the checklist, standard library imports should come before local application imports. Consider moving the `import datetime` statement above the `from app.errors import ...` statement.